scripts/add-unreleased-section.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+die() {
+  local exit_code=1
+  local OPTIND=1
+  local opt
+
+  while getopts "c:" opt; do
+    case "$opt" in
+      c)
+        exit_code="$OPTARG"
+        ;;
+    esac
+  done
+
+  shift $((OPTIND - 1))
+
+  echo "ERROR:" "$@" >&2
+  exit $exit_code
+}
+
+print_usage() {
+  cat <<EOF
+Usage: ${0##*/} <package>
+       ${0##*/} -h
+Adds an unreleased section to the CHANGELOG.md for <package>.
```

Comment:
Add note to the effect of: This script must be executed from the root of the `storefront` repository.

Reply:
Line 50 takes care of that, so it can be called from anywhere.

Reply:
Noted, resolving.